t/admin/consumers.t

```diff
@@ -87,7 +87,7 @@ passed
                         "desc": "new consumer",
                         "plugins": {
                             "key-auth": {
-                                "key": "auth-one"
+                                "key": "4y+JvURBE6ZwRbbgaryrhg=="
```

Comment:
since data encryption is `true` now, the body returned after a `PUT` operation would consist of encrypted information.